src/lib/core/CHIPPersistentStorageDelegate.h

```diff
@@ -97,6 +87,25 @@ class DLL_EXPORT PersistentStorageDelegate
      *         or another CHIP_ERROR value from implementation on failure.
      */
     virtual CHIP_ERROR SyncDeleteKeyValue(const char * key) = 0;
+
+    /**
+     * @brief
+     *   Helper function that identifies if a key exists.
```

Comment:
```suggestion
     *   Helper function that identifies if a key exists.
     *
     *   This may be overridden to provide an implementation that is simpler or more direct.     
```